src/modules/assets/assets.controller.ts

```diff
@@ -20,10 +22,11 @@ import { Order } from './assets.types';
 @Auth()
 @UseInterceptors(SentryErrorInterceptor)
 @ApiTags('Assets')
-@Controller('assets')
+@Controller({ path: 'assets', version: VERSION_NEUTRAL })
 export class AssetsController {
   constructor(private readonly assetsService: AssetsService) { }
 
+  @Version('1')
```

Comment:
Why only this route is version 1?

Reply:
sorry, this was for testing purposes, removed.